fitbert_test_data/README.md

```diff
@@ -0,0 +1,71 @@
+# Fill-in-the-blanks dataset (test data for FitBERT)
+
+<em> "FitBert ((F)ill (i)n (t)he blanks, (BERT)) is a library for using [BERT](https://arxiv.org/abs/1810.04805) to fill in the blank(s) in a section of text from a list of options. "</em>
+
+Every solution needs a test set. This subrepo is meant to generate fill-in-the-blanks questions out of any (reasonable) plain text.
+
+
+## Datasets
+
+Plan text (used to generate fill-in-the-blank tasks) comes from:
+
+[Kaggle: Global News Dataset](https://www.kaggle.com/dbs800/global-news-dataset)
+
+You can use other datasets in `.csv` format, but remember to rename text column to `text`.
+
+## Installation:
+
+- Dependencies:
+`pip3 install -r requirements.txt`
+
+- Spacy default model:
+`python3 -m spacy download en_core_web_sm`
+
+- Wordnet for NLTK: 
+```python
+import nltk
+nltk.download()
+```
+
+## Usage
```

Comment:
Is there a command that we could put in this section that will work out of the box?

Reply:
Nice readme fix, thx!